SECURITY.md

```diff
@@ -11,10 +11,10 @@ unless this is not possible or feasible with a reasonable effort.
 
 | Version | Supported          |
 | ------- | ------------------ |
+| 7.4.x   | :white_check_mark: |
 | 7.2.x   | :white_check_mark: |
 | 7.0.x   | :white_check_mark: |
-| 6.2.x   | :white_check_mark: |
-| < 6.2   | :x:                |
```

Comment:
AFAICT that's wrong.
see https://redis.io/about/releases/
6.2 should still be supported, and we should drop support for 7.0.
@yossigo please ack.

Reply:
Fixed

Reply:
@yossigo @YaacovHazan PTAL and ack.